docs/configuration/OVERVIEW.md

```diff
@@ -36,103 +44,153 @@ A component will have to handle different types of configurations.
 
 ### Default configuration
 
-- Each __component type__ will have a default configuration which will be defined in _.config.ts_ file associated to the
-  component
+- Each __configurable component__ will have a default configuration which will be defined in the `.config.ts` file associated to the component
 
-> __WARNING__ the field name 'id' should not be used in the configuration, as we created a unique one for the entity configuration store
+> [!WARNING]
+> The field name `id` should not be used in the configuration, as we use this field in our internal implementation.
```

Comment:
You may create a linter rule for that ! :)

Reply:
We can do it in a separate Github issue ;)